modules/functional_expansion_tools/include/bcs/FEFluxBC.h

```diff
@@ -0,0 +1,34 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef FEFLUXBC_H
+#define FEFLUXBC_H
+
+// MOOSE includes
+#include "FunctionNeumannBC.h"
+
+// Forward declarations
+class FEFluxBC;
+
+template <>
+InputParameters validParams<FEFluxBC>();
+
+/// Defines an FE-based BC that strongly encourages the gradients to match
```

Comment:
Class description and methods use:
```
/**
 * Comment
 */
```

Member variables use:
```
/// comment
```

Reply:
Good to know.

I used the triple slash for single-line and the stars for multi-line. It will take a while, but I'll get the refactoring done for this.